community/radare2.hpkg

```diff
@@ -0,0 +1,105 @@
+                               [core/coreutils
+                                core/findutils
+                                core/gcc
+                                core/grep
+                                core/make
+                                core/patch
+                                core/sed
+                                core/awk
+                                core/tar
+                                core/pkgconf
+                               ]))
+    (core/link-/bin/sh)
+    (unpack-src radare2-src)
+    (os/setenv "CFLAGS"
+               (string *default-cflags*
+                       " "
+                       (string/join (map |(string "-I" ($ :path) "/include")
+                                         [capstone
+                                          file
+                                          zlib
+                                          lz4
+                                          xxhash
+                                          openssl
+                                          libuv
+                                          libzip]) " ")
+                       " "
+                       (sh/$<_ pkg-config --cflags capstone "liblz4" zlib libuv libxxhash openssl)))
+    (os/setenv "LDFLAGS"
+               (string *default-ldflags*
+                       " "
+                       (string/join (map |(string "-L" ($ :path) "/lib")
+                                         [capstone
+                                          file
+                                          zlib
+                                          lz4
+                                          xxhash
+                                          openssl]) " ")
+                       " "
+                       (string/join (map |(string "-L" ($ :path) "/lib64")
+                                         [libuv
+                                          libzip]) " ")
+                       " "
+                       "-Wl,-rpath=" (string
+                                      (dyn :pkg-out) "/lib" # to find own libs
+                                      ":"
+                                      (join-pkg-paths ":" "/lib"
+                                                      [capstone file zlib lz4 xxhash openssl])
+                                      ":"
+                                      (join-pkg-paths ":" "/lib64"
+                                                      [libuv libzip]))
+                       " "
+                       "-lmagic" # doesn't have a .pc
+                       " "
+                       (sh/$<_ pkg-config --libs capstone zlib "liblz4" libxxhash openssl libuv libzip)))
+
+    (sh/$ ./configure
+          --prefix= ^ (dyn :pkg-out)
+          --with-syscapstone
+          --with-sysxxhash
+          --with-syszip
+          --with-sysmagic
+          --with-openssl)
+
+    # XXX: seems like it the makefiles assumes that /lib exists
+    (os/mkdir (string (dyn :pkg-out) "/lib"))
```

Comment:
I tried building without this line and didn't have any difficulties -- that is, I looked at the produced result and noticed that there was a populated `lib` directory.

May be that didn't occur for you?

Reply:
You're correct. I think that what I saw was an error caused by something else, which is not there anymore.